releasenotes/notes/diagnose-info-log-ae1b2adf9a60ae58.yaml

```diff
@@ -0,0 +1,5 @@
+---
+enhancements:
+  - |
+    The ``agent diagnose`` and ``agent flare`` commands no longer create error-level log messages when the diagnostics fail.
```

Comment:
I am not 100% sure, but we use pandoc to generate markdown from the RST, and we then use that on GH releases, the double back-ticks might cause some issues. Definitely something we can find out when the time comes.

Reply:
I think ``..`` is [proper rST syntax](https://docutils.sourceforge.io/docs/user/rst/quickstart.html) -- it seems if pandoc is doing its job, it should work?  Let's find out :)

Reply:
(note: I saw some other double-backticks in other PRs, so I'm pretty sure this is OK)